test/modules/paymentProcessor/bridging/abstracts/Interfaces/IEverClearSpoke.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
```

Comment:
SHould be in folder `test/modules/paymentProcessor/Interfaces/IEverClearSpoke.sol`

Reply:
Only now seeing that it isn't used I believe